### PR TITLE
Change "magazines" section to "newsletters" and update resources at library page

### LINKED ISSUE #1976: Improve Magazines section of the Library page
The current Magazines section lists two "Revuo Monero" editions from 2017.

I believe the section could be improved by removing the two links and add links to the Revuo website (https://revuo-xmr.com/), to 
 The Monero Moon (https://www.themoneromoon.com/), and maybe to Monero Outreach (https://www.monerooutreach.org/).

### PR DESCRIPTION
**Pull request by HardenedSteel**
* Renamed "magazines" title because using "newsletters" title is more appropriate. 
* Didn't write a description for resources, if needed feel free to make suggestion.  
* Didn't include [revuo-xmr.com](https://revuo-xmr.com/) because @erciccione pointed some issues at [#1976](https://github.com/monero-project/monero-site/issues/1976#issuecomment-1207891953)

and it will close #1976

### CODE REVIEW

**Comment by plowsof** [comment]
If the [moneromoons ccs proposal](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/336) is not merged to funding , he will not produce the newsletter anymore , something to consider (and is currently away for the next 4 weeks)

**Review by erciccione — CHANGES_REQUESTED** [review]
Better add a brief description of each newsletter otherwise it's quite ugly and differs from the style of the rest of the page.

**Comment by erciccione** [comment]
Opinions about my comment above?

**Comment by erciccione** [comment]
@HardenedSteel i don't know as i'm not very familiar with any of them, but even just author and structure (published every N) would be enough.

File: _i18n/en.yml [hunk 1]
```diff
@@ -907,15 +907,14 @@ library:
...
-
-moneropedia:
+  newsletters: Newsletters
+  themonerostandard: >
+    Monero Standard is a weekly publication by recanman, providing up-to-date news and information on the Monero protocol.
+    In addition to covering the latest developments in the Monero community, The Monero Standard also includes a weekly price chart, mining pool chart, blockchain statistics, and even a Meme of the Week.
+  moneroobserverblitz: >
+    Monero Observer Blitz provides monthly updates on the latest developments in the Monero community and protocol.
+    Written by escapethe3RA, this publication covers news related to Monero's ongoing development, community events, and notable achievements within the space.
+  moneropedia:
```
[the remark this answers is not in the document]
Comment (HardenedSteel) [reply _i18n/en.yml]: Thank you!

File: library/index.md [hunk 2]
```diff
@@ -43,12 +43,12 @@ permalink: /library/index.html
...
     <section class="container full">
         <div class="info-block text-adapt">
-            <h2>{% t library.magazines %}</h2>
+            <h2>{% t library.newsletters %}</h2>
             <div>
-                <h3><a href="{{ site.baseurl_root }}/library/Revuo-2017-Q4.pdf">Revuo Monero Q4 2017</a></h3>
-                    <p>{% t library.revuoq4p %}</p>
-                <h3><a href="{{ site.baseurl_root }}/library/Monero-Revuo-3Q-2017.pdf">Revuo Monero Q3 2017</a></h3>
-                    <p>{% t library.revuoq3p %}</p>
+                <h3><a href="https://localmonero.co/the-monero-standard">The Monero Standard Series</a></h3>
```
Comment (erciccione) [inline library/index.md:48]: Here we should either make `series` translatable, or remove it all together and leave only `The Monero Standard`

File: library/index.md [hunk 3]
```diff
@@ -43,12 +43,12 @@ permalink: /library/index.html
...
+                  <p>{% t library.themonerostandard %}</p>
+                <h3><a href="https://monero.observer/tag/blitz/">Monero Observer Blitz Series</a></h3>
```
Comment (erciccione) [inline library/index.md:50]: Same as above: should be `Monero Observer Blitz`, or `series` should be translatable.

**Review by erciccione — CHANGES_REQUESTED** [review]

**Comment by erciccione** [comment]
ping @HardenedSteel

**Review by erciccione — APPROVED** [review]

**Comment by plowsof** [comment]
we can merge as is OR add/test html scripting with this pull request ive made here https://github.com/HardenedSteel/monero-site/pull/2

**Comment by erciccione** [comment]
I'd merge as it is, since this is already reviewed and i would open a dedicated pr after if you want to propose that patch. That said, personally i don't see the need to increase complexity and add a new file. It's only two entries, but even if it was more, the current structure could still be used without problems.

**Pull request merged**